Cargo.toml

```diff
@@ -34,6 +34,7 @@ lazy_static = "0.1.15"
 regex = "0.1.71"
 rustc-serialize = "0.3"
 clippy-mini-macro-test = { version = "0.1", path = "mini-macro" }
+serde = "0.7"
```

Comment:
Oh, sorry, I didn't see it was in `[dev-dependencies]` because GitHub folded code just before it :confused:
I'm ok with this then. But since we've had problems depending on `regex_macro` in the past, let's see what other have to say about that.